packages/playground/src/domains/workflows/workflow/__tests__/use-workflow-trigger.test.tsx

```
@@ -0,0 +1,154 @@
+// @vitest-environment jsdom
+import type { GetWorkflowResponse } from '@mastra/client-js';
```

ESLint

[error] '`@mastra/client-js`' imported multiple times.

(import/no-duplicates)